e2e/helpers/testUser.ts

```diff
@@ -71,10 +71,26 @@ export async function loginTestUser(
 
   await page.getByRole('link', { name: 'Log in to Synapse' }).first().click()
   await page.getByRole('button', { name: 'Sign in with your email' }).click()
-  await page.getByLabel('Username or Email Address').fill(testUserName)
-  await page.getByLabel('Password').fill(testUserPassword)
+
+  const usernameInput = page.getByLabel('Username or Email Address')
+  await usernameInput.fill(testUserName)
+  await expect(usernameInput).toHaveValue(testUserName)
+
+  const passwordInput = page.getByLabel('Password')
+  await expect(passwordInput).toBeEmpty()
+  await passwordInput.fill(testUserPassword)
+  await expect(passwordInput).not.toBeEmpty()
+
   await page.getByRole('button', { name: 'Sign in' }).click()
 
+  // Ensure that correct username/password were received
+  const loadingButton = page.getByRole('button', { name: 'Logging you in' })
+  await expect(loadingButton).toBeVisible()
+  await expect(loadingButton).not.toBeVisible()
+  await expect(
+    page.getByText('The provided username/password combination is incorrect'),
+  ).not.toBeVisible()
```

Comment:
Confirm that the correct username/password were received, hopefully will help us troubleshoot if login fails to redirect as in [this run](https://github.com/Sage-Bionetworks/SynapseWebClient/actions/runs/6123409526/job/16621237584)